release-notes/SupportedFeatures-v2.1.0.md

```diff
@@ -0,0 +1,63 @@
+|Client Authentication|Basic           |:white_check_mark:                  |:white_check_mark:                         | :white_check_mark:                  |
+|              |Digest                 |:white_check_mark:                  |:white_check_mark:                         | :white_check_mark:|
+|              |NTLM                   |:white_check_mark:                  |[:no_entry_sign:](https://github.com/dotnet/wcf/releases/tag/v2.0.0)           | [:no_entry_sign:](https://github.com/dotnet/wcf/releases/tag/v2.0.0)    |
+|              |Kerberos/TCP           |:white_check_mark:                  |:white_check_mark:                         | :white_check_mark:                  |
+|              |Kerberos/HTTP          |:white_check_mark:                  |:white_check_mark:           | :white_check_mark:    |
+|              |Certificate/TCP       |:white_check_mark:                  |:white_check_mark:                         | :white_check_mark:                  |
+|              |Certificate/HTTP      |:white_check_mark:                  |:white_check_mark:                         | :white_check_mark:                      |:white_check_mark: 
+|              |Certificate/PeerTrust      |:white_check_mark:                  |:white_check_mark:                         | [:no_entry_sign:](https://github.com/dotnet/wcf/releases/tag/v2.0.0)                     |
+|Service Authentication|Certificate    |:white_check_mark:                  |:white_check_mark:                         | :white_check_mark:                  |
+|              |SPNIdentity/TCP    |:white_check_mark:          |:white_check_mark:                  | :white_check_mark:          |
+|              |SPNIdentity/HTTP    |[:no_entry_sign:](https://github.com/dotnet/wcf/releases/tag/v2.0.0)          |[:no_entry_sign:](https://github.com/dotnet/wcf/releases/tag/v2.0.0)                  | [:no_entry_sign:](https://github.com/dotnet/wcf/releases/tag/v2.0.0)          |
+|              |UPNIdentity    |:white_check_mark:          |[:warning:](https://github.com/dotnet/wcf/releases/tag/v2.0.0)                   | [:warning:](https://github.com/dotnet/wcf/releases/tag/v2.0.0)          |
+|Client types  |ChannelFactory\<T\>    |:white_check_mark:                  |:white_check_mark:                         | :white_check_mark:                  |
+|              |ChannelBase\<T\>       |:white_check_mark:                  |:white_check_mark:                         | :white_check_mark:                  |
+|Proxy Support  |Http&Https    |:white_check_mark:                  |[:warning:](https://github.com/dotnet/wcf/releases/tag/v2.1.0)                   | [:warning:](https://github.com/dotnet/wcf/releases/tag/v2.1.0)          |                  |
```

Comment:
I thought the proxy support for *nix is fixed, no?

Reply:
Since we feel that the feature is not fully tested, we updated the status from not supported to not fully verified.

Reply:
I see. Thanks.